pkg/runtime/wfengine/workflowstate_test.go

```diff
@@ -73,45 +75,43 @@ func TestClearingInbox(t *testing.T) {
 	state.ClearInbox()
 
 	req, err := state.GetSaveRequest("wf1")
-	if assert.NoError(t, err) {
-		assert.Equal(t, "wf1", req.ActorID)
-		assert.Equal(t, workflowActorType, req.ActorType)
+	require.NoError(t, err)
```

Comment:
I wish I'd known about the `require` package sooner. 😅 Thanks for taking the time to clean this up.